redirects.ts

```diff
@@ -189,15 +189,15 @@ export const redirectsOptions: Options = {
             from: '/general/run-a-node/node-operator-overview', // #200 Restructure docs
         },
         {
-            to: '/node/mainnet/',
-            from: '/general/oasis-network/network-parameters', // #200 Restructure docs
+            to: '/node/network-parameters/mainnet',
+            from: '/general/oasis-network/network-parameters', // #200 Restructure docs, #1438 Reorganize network parameters
```

Comment:
Use array in `from` to map multiple old URLs to the same `to` field.

```suggestion
            from: [
                '/general/oasis-network/network-parameters', // #200 Restructure docs
                '/node/mainnet/', // #1438 Reorganize network parameters
            ],
```